### ISSUE TITLE
Most github repo instructions are always incomplete!

### ISSUE THREAD

**Issue by AntDX316** [Stability-AI/generative-models#184]
https://www.youtube.com/watch?v=HMW9hVoQa0M

I also had to do this in between:
pip install torch==2.0.1+cu118 -f https://download.pytorch.org/whl/torch_stable.html

**Comment by zephirusgit** [comment]
Yes, it is really detestable, you have to walk guessing, that they use or not, they do not put a pip list with the versions of things, half of the requirements are always incomplete or not.
I mean, they don't try to try, what they even write.
Apart one has to always create a new environment, so as not to have the PC full of useless or incompatible things, because they are unable to update something without stopping to work.
The truth is a great discomfort, but 5 minutes cannot be taken to say, we rely on such a thing, that such another will require, for this and such another for that, of other people one can limit themselves to thanking the shared, but say, I share This so that they cannot use it, it is ridiculous, if there is any pinch of genius in what they do is directly proportional to how useless they are to explain how it works ..
I thank you for your video, I'm going to look at it and subscribe

**Comment by zephirusgit** [comment]
https://github.com/Stability-AI/generative-models/assets/20031912/d7b29bf4-c764-4f72-9a05-66c61def3621

**Comment by zephirusgit** [comment]
https://github.com/Stability-AI/generative-models/assets/20031912/d6a539db-16f9-460c-b2e9-edac0ab13348

**Comment by AntDX316** [comment]
People going through and helping what didn't exist is how to do anything.
